src/utils/interaction.js

```diff
@@ -1,25 +1,42 @@
-import { items, displayTask } from './structure.js';
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] Kindly remove this eslint disable line its best to fix the issue in the code.